dask/bytes/pyarrow.py

```diff
@@ -32,7 +32,7 @@ class PyArrowHadoopFileSystem(object):
     sep = "/"
 
     def __init__(self, **kwargs):
-        self.fs = pa.hdfs.HadoopFileSystem(update_hdfs_options(**kwargs))
+        self.fs = pa.hdfs.HadoopFileSystem(update_hdfs_options(kwargs))
```

Comment:
Yep, the signature is 
```
pa.hdfs.HadoopFileSystem(host='default', port=0, user=None, kerb_ticket=None, driver='libhdfs', extra_conf=None)
```
So the [current code](https://github.com/dask/dask/blob/master/dask/bytes/pyarrow.py#L35) is correct, and the helper function `update_hdfs_options` takes a dict and returns a dict, which is what `kwargs` is.